packages/atomic/src/components/atomic-breadcrumb-manager/atomic-breadcrumb-manager.tsx

```diff
@@ -23,14 +23,9 @@ import dayjs from 'dayjs';
 /**
  * A component that creates breadcrumbs that display the currently active facet values
  *
- * @part breadcrumbs - Container for all types of breadcrumbs
```

Comment:
I found that there were too many parts here, leading to confusion. I left the crucial ones, but I could add back more